app/container.mac.php

```diff
@@ -16,20 +17,34 @@
     return new ShellCreator();
 };
 
+$container['installer.dns.dnsdock'] = function($c) {
+    return new Dns\Mac\DnsDock(new SshClient(new Session(
+        new Configuration(SshClient::DEFAULT_HOSTNAME),
+        new Password(SshClient::DEFAULT_USERNAME, SshClient::DEFAULT_PASSWORD)
+    )), $c['console.user_interaction'], $c['process.interactive_runner'])
```

Comment:
You are missing the semicolon at the end of the line, which ends to a PHP fatal error ;-)